batch/batch/worker.py

```diff
@@ -354,22 +348,18 @@ async def run(self, worker, cpu_sem):
                 self.container = await docker_call_retry(MAX_DOCKER_OTHER_OPERATION_SECS, f'{self}')(
                     create_container, config, name=f'batch-{self.job.batch_id}-job-{self.job.job_id}-{self.name}')
 
-            async with cpu_sem(self.cpu_in_mcpu, f'{self}'):
-                async with self.step('runtime', state=None):
-                    if self.name == 'main':
-                        asyncio.ensure_future(worker.post_job_started(self.job))
-
-                    async with self.step('starting'):
-                        await docker_call_retry(MAX_DOCKER_OTHER_OPERATION_SECS, f'{self}')(
-                            start_container, self.container)
+            async with self.step('runtime', state=None):
```

Comment:
So I believe runtime was the step we used to calculate costs from, since it was the step that coincided with holding the cpu semaphore.  I think it can go away now.